components/report/identity.tsx

```diff
@@ -6,29 +6,30 @@ import { AccountHolderQuery } from '../../integrations/stitch/query/queries';
 export function Identity(): JSX.Element {
     const accountHolderQuery = useQuery<AccountHolderResponse>(AccountHolderQuery);
     const identity = accountHolderQuery.data?.user.bankAccounts.find(b => b.accountHolder != null)?.accountHolder;
+    console.log(JSON.stringify(identity));
     if (identity) {
         return (
             <>
                 <article className="media is-align-items-center mb-0 mb-3">
                     <figure className="media-left ml-0">
                         <p className="image is-128x128">
                             <img className="is-rounded"
-                                src={`https://robohash.org/${encodeURIComponent(identity.fullName)}`}
+                                src={`https://robohash.org/${encodeURIComponent(identity.fullName ?? identity.name)}`}
                                 alt="Placeholder image"/>
                         </p>
                     </figure>
                     <div className="media-content">
                         <div className="content has-text-weight-light is-large">
                             <p>
                                 <span className="is-size-6">Earth Name</span><br/>
-                                <strong className="">{identity.fullName}</strong>
+                                <strong className="">{identity.fullName ?? identity.name}</strong>
                             </p>
                         </div>
                     </div>
                 </article>
                 <IdentityTableRow heading="Communicator" value={identity.contact?.phoneNumber}/>
                 <IdentityTableRow heading="Email" value={identity.email}/>
-                <IdentityTableRow heading="Identifier" value={identity.identifyingDocument?.number}/>
+                <IdentityTableRow heading="Identifier" value={identity.identifyingDocument?.number ?? identity.registrationNumber}/>
```

Comment:
Should we have a different heading attribute for an individual's id number vs a company's registration number?